singularity/code/screens/map.py

```diff
@@ -425,7 +425,7 @@ def save_game(self):
                                                "Are you sure to overwrite the saved game ?"))
                 overwrite = dialog.call_dialog(yn, self)
                 if not overwrite:
-                    return
+                    self.save_game()
```

Comment:
Love the idea. I think we need a more general implementation though because we will likely need it again elsewhere at some point.

Reply:
What exactly do you have in mind here? I'd be happy to change it, but I don't know yet where you wish to go with this.

Reply:
Ah, I checked the code; and I thought the context was different.  If anything, we would either have to rewrite it to a proper dialog (rather than a function that calls a dialog or two) or use a loop until we get a result (or cancel).

But both can happen in a separate PR.